src/editors/TextEditor/TextEditor.jsx

```diff
@@ -27,24 +46,27 @@ const TextEditor = () => {
         )
         : (
           <Editor
-            onInit={(evt, editor) => { editorRef.current = editor; }}
+            onInit={(evt, editor) => {
+              editorRef.current = editor;
+            }}
             initialValue={blockValue ? blockValue.data.data : ''}
             init={{
-              height: '100%',
+              setup: (editor) => {
+                editor.ui.registry.addButton('imageuploadbutton', {
+                  icon: 'image',
+                  onAction: () => openUploadModal(),
+                });
+              },
+              plugins: 'link codesample emoticons table charmap code autoresize',
```

Comment:
Nit: The nit above also applies to this plugin list.

Reply:
Well, we will specify the required plugins as props to the paragon component.